mm/iob/iob_initialize.c

```diff
@@ -52,11 +51,13 @@
  */
 
 #ifdef IOB_SECTION
-static uint8_t g_iob_buffer[IOB_BUFFER_SIZE] locate_data(IOB_SECTION);
+static uint8_t g_iob_buffer[IOB_POOLSIZE] locate_data(IOB_SECTION);
 #else
-static uint8_t g_iob_buffer[IOB_BUFFER_SIZE];
+static uint8_t g_iob_buffer[IOB_POOLSIZE];
 #endif
 
+static struct iob_s g_iob_list[CONFIG_IOB_NBUFFERS];
```

Comment:
g_iob_list->g_iob_pool

Reply:
Done ,move to https://github.com/apache/nuttx/pull/7920